### PR TITLE
Update PDP-6 and TV-11 simulators.

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
Update two submodules.  The TV-11 is now in another GitHub repository.

### CODE REVIEW

**Comment by eswenson1** [comment]
@larsbrinkhoff  Why did you assign this to me?

**Comment by larsbrinkhoff** [comment]
Sorry, I clicked the wrong place.  I meant review, not assign.

File: Makefile [hunk 1]
```diff
@@ -217,7 +217,7 @@ $(MAGFRM):
 	$(MAKE) -C tools/dasm
 
 $(TV11):
-	$(MAKE) -C tools/tv11
+	$(MAKE) -C tools/tv11 CFLAGS=-O3
```
Comment (larsbrinkhoff) [inline Makefile]: @aap, I had to do this to work around an old GCC which doesn't understand -fno-diagnostics-color and -fno-diagnostics-show-caret.

**Comment by larsbrinkhoff** [comment]
I think @aap has some update for the PDP-6 simulator what would be nice to include here, so I'll wait some more to merge this.

CircleCI failing builds are due to it not seeing updated submodule commit IDs.  I have seen this before.

**Comment by larsbrinkhoff** [comment]
Checked that .; @ SPCWAR and MUNCH are working.

**Pull request merged**